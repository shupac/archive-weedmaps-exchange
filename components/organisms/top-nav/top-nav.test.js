import React from 'react';
import { shallow } from 'enzyme';
import TopNav from './';

describe('Top Nav', () => {
  it('Top Nav', () => {
    const tree = shallow(
      <TopNav
        activeLink="deals"
        avatarUrl="http://localhost"
        onMenuClick={() => null}
        count={5}
        userName="gabOng"
      />,
    );
    expect(tree).toMatchSnapshot();
  });
});
