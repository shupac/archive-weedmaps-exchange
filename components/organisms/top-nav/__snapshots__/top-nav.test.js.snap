// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Top Nav Top Nav 1`] = `
<top-nav__TopNavContainer>
  <top-nav__LeftContainer>
    <top-nav__MenuButton
      onClick={[Function]}
    >
      <Menu />
    </top-nav__MenuButton>
    <Heading
      active="deals"
    />
  </top-nav__LeftContainer>
  <top-nav__RightContainer>
    <UserDropdown
      avatarUrl="http://localhost"
      user=""
    />
  </top-nav__RightContainer>
</top-nav__TopNavContainer>
`;
